make_qlf_k4n8_default_fasm.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+if [ -z "${QLF_FASM_DB_ROOT}" ]; then
+    echo "ERROR: Please set the env. var. QLF_FASM_DB_ROOT to point to FASM database"
+    exit -1
+fi
+
+set -e
+
+python3 -m qlf_fasm.make_default_fasm --log-level DEBUG --db-root ~/Repos/qlfpga-symbiflow-plugins/qlf_k4n8/fasm_database --features "\
+    logical_tile_io_mode_io__0.logical_tile_io_mode_physical__iopad_0.logical_tile_io_mode_physical__iopad_mode_default__pad_0.IO_QL_CCFF_mem.mem_out,\
+    logical_tile_io_mode_io__1.logical_tile_io_mode_physical__iopad_0.logical_tile_io_mode_physical__iopad_mode_default__pad_0.IO_QL_CCFF_mem.mem_out,\
+    logical_tile_io_mode_io__2.logical_tile_io_mode_physical__iopad_0.logical_tile_io_mode_physical__iopad_mode_default__pad_0.IO_QL_CCFF_mem.mem_out,\
+    logical_tile_io_mode_io__3.logical_tile_io_mode_physical__iopad_0.logical_tile_io_mode_physical__iopad_mode_default__pad_0.IO_QL_CCFF_mem.mem_out,\
+    logical_tile_io_mode_io__4.logical_tile_io_mode_physical__iopad_0.logical_tile_io_mode_physical__iopad_mode_default__pad_0.IO_QL_CCFF_mem.mem_out,\
+    logical_tile_io_mode_io__5.logical_tile_io_mode_physical__iopad_0.logical_tile_io_mode_physical__iopad_mode_default__pad_0.IO_QL_CCFF_mem.mem_out,\
+    logical_tile_io_mode_io__6.logical_tile_io_mode_physical__iopad_0.logical_tile_io_mode_physical__iopad_mode_default__pad_0.IO_QL_CCFF_mem.mem_out,\
+    logical_tile_io_mode_io__7.logical_tile_io_mode_physical__iopad_0.logical_tile_io_mode_physical__iopad_mode_default__pad_0.IO_QL_CCFF_mem.mem_out,\
+    logical_tile_io_mode_io__8.logical_tile_io_mode_physical__iopad_0.logical_tile_io_mode_physical__iopad_mode_default__pad_0.IO_QL_CCFF_mem.mem_out,\
+    logical_tile_io_mode_io__9.logical_tile_io_mode_physical__iopad_0.logical_tile_io_mode_physical__iopad_mode_default__pad_0.IO_QL_CCFF_mem.mem_out,\
+    logical_tile_io_mode_io__10.logical_tile_io_mode_physical__iopad_0.logical_tile_io_mode_physical__iopad_mode_default__pad_0.IO_QL_CCFF_mem.mem_out,\
+    logical_tile_io_mode_io__11.logical_tile_io_mode_physical__iopad_0.logical_tile_io_mode_physical__iopad_mode_default__pad_0.IO_QL_CCFF_mem.mem_out,\
+    logical_tile_io_mode_io__12.logical_tile_io_mode_physical__iopad_0.logical_tile_io_mode_physical__iopad_mode_default__pad_0.IO_QL_CCFF_mem.mem_out,\
+    logical_tile_io_mode_io__13.logical_tile_io_mode_physical__iopad_0.logical_tile_io_mode_physical__iopad_mode_default__pad_0.IO_QL_CCFF_mem.mem_out,\
+    logical_tile_io_mode_io__14.logical_tile_io_mode_physical__iopad_0.logical_tile_io_mode_physical__iopad_mode_default__pad_0.IO_QL_CCFF_mem.mem_out,\
+    logical_tile_io_mode_io__15.logical_tile_io_mode_physical__iopad_0.logical_tile_io_mode_physical__iopad_mode_default__pad_0.IO_QL_CCFF_mem.mem_out"
```

Comment:
Can this be generated somehow? Or maybe defined in an external file and read from there

Reply:
Hmm, The actual bit name is unique but the only difference between them is a single number. I can make a for loop that generates all the names from a single template.

Reply:
Ok, fixed.